### PR TITLE
Config to switch OpenHPC home directory to storage-ssd

### PR DESCRIPTION
**Pull request by brtknr**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ansible>=2.9.0
 os-client-config
 shade
-openstacksdk
+openstacksdk<=0.45.0; python_version<='2.7'
```
Comment (sjpb) [inline requirements.txt:4]: openstacksdk [release notes](https://docs.openstack.org/releasenotes/openstacksdk/ussuri.html#relnotes-0-46-0-stable-ussuri) state:
> openstacksdk does not test or support python2 as of 0.40, but the releases have still accidentally worked (except for 0.44 which was broken for python2).

So should this be either `==0.40.0` (strictly) or at least `==0.45.0` given we must be fairly confident that works?

File: ansible/manila-client.yml [hunk 2]
```diff
@@ -4,9 +4,6 @@
 
 # Query OpenStack Manila for details aob
 - hosts: openstack
-  pre_tasks:
```
Comment (sjpb) [inline ansible/manila-client.yml:7]: Should `ceph_mount_share_name: HomeDirs` get removed from `ansible/group_vars/all/alaska` too then? I know it won't actually hurt but will be confusing.

File: ansible/requirements.yml [hunk 3]
```diff
@@ -12,10 +12,11 @@
 - src: stackhpc.beegfs
   version: v19.8.2
 - src: stackhpc.monasca-agent
+  version: 1.0.4
```
Comment (sjpb) [inline ansible/requirements.yml:15]: Latest on ansible galaxy is 2.0.0 but github seems to be up to 2.10.1. Not sure if we need to bump this more?

File: ansible/requirements.yml [hunk 4]
```diff
@@ -12,10 +12,11 @@
...
 - src: stackhpc.monasca-rsyslog
 - src: stackhpc.grafana-conf
 - src: stackhpc.gluster-cluster
 - src: stackhpc.openhpc
-  version: v0.4.0
+  version: v0.5.0
```
Comment (sjpb) [inline ansible/requirements.yml:20]: Looks good.

File: ansible/manila-client.yml (hunk 2 above)
Comment (brtknr) [reply ansible/manila-client.yml:7]: `ceph_mount_share_name` is used by manila-client.yml further down the file. The debug was accidently commited here last time so therefore removing it.

File: ansible/requirements.yml (hunk 3 above)
Comment (brtknr) [reply ansible/requirements.yml:15]: 2.*.* assumes use of python 3, hence contraining the version to 1.0.4.

File: requirements.txt (hunk 1 above)
Comment (brtknr) [reply requirements.txt:4]: I will change this to ==0.45.0, 0.45.0 is the last supported version according to pip-requires.

**Pull request merged**